Replicator/DBAccess.cc

```diff
@@ -234,6 +234,14 @@ namespace litecore { namespace repl {
                 // TODO: Should skip this entry if a blob with the same digest exists
                 enc.writeKey(key);
                 enc.writeValue(i.value());
+                {
+                    // CBL-4944
+                    if (Value v = i.value(); v) {
+                        if (Dict d = v.asDict(); d && !d.get("length"_sl)) {
+                            warn("Find attachment with nil length in DBAccess::encodeRevWithLegacyAttachments, %s", d.toJSONString().c_str());
```

Comment:
CBSE case has attachment length = 0 while it shouldn't be 0. The attachment length = 0 could be valid for empty blob but for the purpose of the investigation, we should log as well.